### ISSUE TITLE
www.ospc.org/ prepended to links from flatblocks

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/webapp-public#70]
In the tax-brain-get-started-blurb flatblock I added hyperlinks to various Github.org sites. When I click on the content of this flatblock from ospc.org/taxbrain, however, it doesn't do anything and apparently prepends www.ospc.org to the URL. Is this a problem with how I wrote the links into the flatblock, or is it an issue with webapp? 

`
<a href = 'www.github.com/opensourcepolicycenter/taxdata'>www.github.com/opensourcepolicycenter/taxdata</a>`

![image](https://cloud.githubusercontent.com/assets/8301092/10559736/b1c7cd38-74c5-11e5-969c-f7720dd996e2.png)

![image](https://cloud.githubusercontent.com/assets/8301092/10559733/a6b2995a-74c5-11e5-86fc-e84aed15a044.png)

cc @talumbau, @zrisher, @mistakevin

**Comment by talumbau** [comment]
@MattHJensen I think you want:

```
<a href='//www.github.com/opensourcepolicycenter/taxdata'>www.github.com/opensourcepolicycenter/taxdata</a>

```

So pre-pending the `www` with a double forward slash `//`. An example of doing this is here:

https://github.com/OpenSourcePolicyCenter/blog-posts/commit/c04d4a97828ddc1b1e61fbda5f1416b7168b1096

This will properly direct to either http or https depending on how the page is currently loaded.

**Comment by MattHJensen** [comment]
@talumbau, this is an improvement, and I made the change to the live site so you can check it out. Now if I right click and select "open link in new tab", it will take me to the right place. But if I left click on the link still nothing happens.

**Comment by talumbau** [comment]
This is true of the other link in the text as well. Interesting. I wonder if this is a JS thing. Perhaps @zrisher could chime in.

**Comment by zrisher** [comment]
There's definitely a JS error bubbling up from code that's meant to run on links in the submission form. I'll take a look at get a fix together as soon as I can.

**Comment by zrisher** [comment]
The JS error will be fixed via #85.

@MattHJensen you'll probably want to add `target="_blank"` to your link tags so they open in a new window by default - I believe that is expected behavior for external links.

This got me thinking about how it would be nice to have flatblocks in development that correspond with the needs of the production setup, so I opened #86.

**Comment by talumbau** [comment]
closed via #85 but the flatblock on the site contains the text "http//" instead of "http://" so one of the links is bad. Should be a simple fix in the flatblock editor. cc @MattHJensen

**Comment by MattHJensen** [comment]
Thanks @zrisher!

> you'll probably want to add target="_blank"

got it. 

> but the flatblock on the site contains the text "http//" instead of "http://" so one of the links is bad

got it.
